examples/jenkins/Linux.Jenkinsfile

```
@@ -77,21 +74,22 @@ pipeline {
       }
     }
     stage('Run Regression') {
+      when { 
+        expression { return params.ApplicationScope != params.ApplicationScopeWithTests } 
+      }
       steps {
-        withPythonEnv('python') {
+        withPythonEnv('python3') {
           echo 'Generating URLs for BDD testing...'
-          // Generate the URL endpoints of the BDD tests
           sh "python3 -m outsystems.pipeline.generate_unit_testing_assembly --artifacts \"${env.ArtifactsFolder}\" --app_list \"${params.ApplicationScopeWithTests}\" --cicd_probe_env ${env.ProbeEnvironmentURL} --bdd_framework_env ${env.BddEnvironmentURL}"
           echo "Testing the URLs and generating the JUnit results XML..."
-          // Run those tests and generate a JUNIT test report
           sh(script: "python3 -m outsystems.pipeline.evaluate_test_results --artifacts \"${env.ArtifactsFolder}\"", returnStatus: true)
         }
       }
       post {
         always {
-          withPythonEnv('python') {
+          withPythonEnv('python3') {
             echo "Publishing JUnit test results..."
-            junit(testResults: "${env.ArtifactsFolder}/junit-result.xml", allowEmptyResults: true)
+            junit(testResults: "${env.ArtifactsFolder}\\junit-result.xml", allowEmptyResults: true)
```

Being a Linux template, maybe we should use a '/' instead here?
```suggestion
            junit(testResults: "${env.ArtifactsFolder}/junit-result.xml", allowEmptyResults: true)
```

---

You're right! template updated